resources/pybind11_2.10.0.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/include/pybind11/cast.h b/include/pybind11/cast.h
+index a0e32281..f132d096 100644
+--- a/include/pybind11/cast.h
++++ b/include/pybind11/cast.h
+@@ -657,7 +657,7 @@ public:
+     }
+ 
+     static constexpr auto name
+-        = const_name("Tuple[") + concat(make_caster<Ts>::name...) + const_name("]");
++        = const_name("Tuple[") + detail::concat(make_caster<Ts>::name...) + const_name("]");
```

Comment:
```suggestion
+        = const_name("Tuple[") + pybind11::detail::concat(make_caster<Ts>::name...) + const_name("]");
```
This seems more robust to me since the other `concat` also lies in a `detail` namespace.

Reply:
Good point, thanks.